tests/test_switch.py

```diff
@@ -594,7 +594,7 @@ def increased_color_temp():
 
 async def test_auto_reset_manual_control(hass):
     switch, (light, *_) = await setup_lights_and_switch(
-        hass, {CONF_AUTORESET_CONTROL: 0.1}
+        hass, {CONF_AUTORESET_CONTROL: 0.2}
```

Comment:
The 0.1 is important because:
```
    # Do a couple of quick changes and check that light is not reset
    for i in range(3):
        _LOGGER.debug("Quick change %s", i)
        await turn_light(True, brightness=(i + 1) * 20)
        await asyncio.sleep(0.05)  # Less than 0.1
        assert manual_control[light.entity_id]
```

Here we do a new operation every 0.05 seconds. So on the third iteration (0.15 s later) the full 0.1 seconds have passed.

If this test doesn't pass on your env, then something in our code must be broken :0